src/cli/rustup_mode.rs

```diff
@@ -1550,16 +1579,16 @@ fn doc(cfg: &Cfg, m: &ArgMatches<'_>) -> Result<utils::ExitCode> {
     }
     let topical_path: PathBuf;
 
-    let doc_url = if let Some(topic) = m.value_of("topic") {
+    let doc_url = if let Some(topic) = m.get_one::<String>("topic") {
         topical_path = topical_doc::local_path(&toolchain.doc_path("").unwrap(), topic)?;
         topical_path.to_str().unwrap()
-    } else if let Some((_, _, path)) = DOCS_DATA.iter().find(|(name, _, _)| m.is_present(name)) {
+    } else if let Some((_, _, path)) = DOCS_DATA.iter().find(|(name, _, _)| m.contains_id(name)) {
```

Comment:
Oops, this should be `.get_flag(name)` (as `DOCS_DATA` arguments are flags thanks to `.action(ArgAction::SetTrue)`), but it doesn't really matter too much.